release-notes/release-notes-v10.7.2.md

```diff
@@ -28,9 +28,19 @@ API breaking changes:
 - Clean up blocks without ledger hash before starting catch-up
 - Don't inject details before publishing events with ZMQ
 - Populate `address_events` also with UTXO events (attach, detach and move)
+- Fix `compose_movetoutxo` documentation
+- Fix `block.close_block_index` field type
+- Set `issuances.reset` and `issuances.locked` default value to False instead None 
 
 ## Codebase
 
+- Replace `counterparty.api.db` with `counterparty.state.db`
```

Comment:
Can we call this `counterparty-state.db`? The double period causes problems for me in some cases.

Also, this will respect `--data-dir`, right? I ran it once and had a problem with that but I'm not sure what happened.

Reply:
or just `state.db`

Reply:
done